### PR TITLE
Use only sysfs for iscsi device attributes

### PR DESCRIPTION
**Pull request by rvykydal**
The changes are required by https://github.com/rhinstaller/anaconda/pull/1960

### CODE REVIEW

File: blivet/udev.py [hunk 1]
```diff
@@ -790,6 +790,16 @@ def device_get_iscsi_name(info):
     return "-".join(path_components[name_field:len(path_components) - 2])
 
 
+def device_get_iscsi_lun(info):
+    lun = 0
+    path_components = device_get_path(info).split("-")
+    idx = list(reversed(path_components)).index("lun")
+    if idx > 0:
+        lun = path_components[-idx]
```
Comment (vpodzime) [inline blivet/udev.py]: Would be nice to have an example string (path) for this magic.

Comment (rvykydal) [reply blivet/udev.py]: The example is a few functions above (common for all the iscsi related functions).

**Comment by rvykydal** [comment]
Updated the unit tests with the new attributes.

**Comment by rvykydal** [comment]
@vpodzime Yes, I'd like to look at what could be moved from blivet/iscsi.py and the credentials are a good candidate. They would be stored in anaconda iscsi module (if we really need them, I have to check).

**Comment by rvykydal** [comment]
It would be great to have this merged and built rather sooner as we have a dependent PR we'd like to merge soon in anaconda.

**Pull request merged**